### PR TITLE
#168131401 Setup Sequelize

### PR DESCRIPTION
**Pull request by drayzii**
#### What does this PR do?
Setup Sequelize
#### Description of Task to be completed?
- Set up Sequelize to work as ORM
- Create a sample user table by generating model and migrations
- Add setup steps to `README.md`
#### How should this be manually tested?
- Install `sequelize` and `sequelize-cli` globally
- For development, we are using a `DEV_DATABASE_URL` variable to keep our DB URL in `.env`  (Add `export` in front of it)
- Then run `npm run db-migrate:dev` to run migrations
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168131401](https://www.pivotaltracker.com/story/show/168131401)
#### Screenshots (if appropriate)
image

#### Questions:
N/A

### CODE REVIEW

**Review by Bahatiroben — COMMENTED** [review]
dude, check the merge conflicts, and make sure you rebase develop because travis is supposed to build.

**Comment by drayzii** [comment]
> dude, check the merge conflicts, and make sure you rebase develop because travis is supposed to build.

I hadn't realised the branch was merged. But recheck it has now been fixed

**Review by j0flintking02 — APPROVED** [review]
LGTM

**Review by Bahatiroben — APPROVED** [review]
looks good now.

File: src/database/config.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const dotenv = require('dotenv');
+
+dotenv.config();
+module.exports = {
+  production: {
+    use_env_variable: 'DATABASE_URL',
+    dialect: 'postgres',
+  },
+  development: {
+    use_env_variable: 'DEV_DATABASE_URL',
```
Comment (Paulvitalis200) [inline src/database/config.js]: I suggest using `process.env` here

File: src/database/config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  production: {
+    use_env_variable: process.env.DATABASE_URL,
+    dialect: 'postgres',
+  },
+  development: {
+    use_env_variable: process.env.DEV_DATABASE_URL,
+    dialect: 'postgres',
+  },
+  test: {
+    use_env_variable: process.env.DEV_DATABASE_URL,
```
Comment (Paulvitalis200) [inline src/database/config.js]: @drayzii Can we have a separate test database instead of using the same Database for tests.

**Review by Paulvitalis200 — APPROVED** [review]

**Pull request merged**